src/common/common_init.cc

```diff
@@ -68,6 +68,8 @@ CephContext *common_preinit(const CephInitParameters &iparams,
       conf->set_val("keyring", "$mds_data/keyring", false);
     else if (conf->name.is_osd())
       conf->set_val("keyring", "$osd_data/keyring", false);
+    else if (conf->name.is_mgr())
```

Comment:
I can move the setting of "keyring" out of here and into ceph_mgr.cc, but we can't set it directly the way Monitor does, because we're using the MonClient machinery for our keyring rather than constructing one by hand.